gazelle/.bazelrc

```diff
@@ -11,3 +11,11 @@ build --incompatible_default_to_explicit_init_py
 # Windows makes use of runfiles for some rules
 build --enable_runfiles
 startup --windows_enable_symlinks
+
+# Do NOT implicitly create empty __init__.py files in the runfiles tree.
+# By default, these are created in every directory containing Python source code
+# or shared libraries, and every parent directory of those directories,
+# excluding the repo root directory. With this flag set, we are responsible for
+# creating (possibly empty) __init__.py files and adding them to the srcs of
+# Python targets as required.
+build --incompatible_default_to_explicit_init_py
```

Comment:
Shouldn't there be an `__init__.py` file in the rules_python_gazelle_helper directory?

Reply:
I have tested with and without and it did not change the behaviour, I think with recent Python versions the `__init__.py` files are optional (see [here](https://docs.python.org/3/whatsnew/3.3.html#pep-420-implicit-namespace-packages)), however, this flag allows us to have better tests.